src/main.ts

```diff
@@ -31,6 +31,22 @@ async function main() {
 
 	const { logger } = Log;
 
+	const publicWsUrls: string[] = [
```

Comment:
How did we gather this list of nodes? 
I think we should at least add a comment here explaining it's a "best effort" list of known public nodes that do not encourage high traffic sidecar installations connecting to them for non-testing/development purposes.

Reply:
@dvdplm  Yea i definitely agree. I didn't know exactly how to reference the actual list itself, but I definitely agree with your description. Thanks I will add that in.